rainbow-me/rainbow
# Make sick haptic feedback on Android

Fixes RNBW-4020
Figma link (if any):

## What changed (plus any additional context for devs)
I created a matching implementation of feedback that is the same on Android as on iOS. I used the newer API so around 10% of devices will be not covered, but this is ok!

 
Some phones (e.g, my OnePlus 7) are not able to make such a short vibration and they need to be very subtle. Then they are just not working and this is OK. Better to now have them than have them too long. 


## Screen recordings / screenshots
no recording, try it. No impact on iOS
## What to test
1. check buttons (selection haptic)
2. copy address (light impact)
3. touch charts (heavy impact)
4. scan QR successfully (notification success)
5. scan unsupported QA (notification error)
6. Press  "Insufficient funds" (notification warning)

They should not be exactly the same. Just soft and expressing more or less what we need 

Please test on some new and popular phones, e.g., samsung s21
